src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -992,6 +1038,11 @@ export class BundleDataClient {
                   }
                 }
               } else {
+                this.logger.debug({
+                  at: "BundleDataClient#loadData",
+                  message: "Duplicate fill detected",
+                  fill,
+                });
```

Comment:
This and the subsequent `Duplicate slow fill request detected` log message are pretty far away from the if statement that can trigger them; it's hard to see the logic in github at least. wdyt about relocating them to the top?

Reply:
creates a big diff but good point: 9e3090bdb9d1926d0569b0a08393bb48362ef59a

Reply:
@pxrl I've decided to revert back to this implementation becaues the diff is so big and hard to verify.